src/java/fr/paris/lutece/portal/service/regularexpression/RegularExpressionService.java

```diff
@@ -79,9 +79,11 @@ public static RegularExpressionService getInstance( )
     {
         if ( _singleton == null )
         {
-            _singleton = new RegularExpressionService( );
+            synchronized ( RegularExpressionService.class )

```

Comment:
here again, synchronisation scope

Reply:
Corrected in another commit of the same PR.